sotodlib/toast/scripts/so_sim.py

```diff
@@ -70,15 +71,21 @@ def simulate_data(job, otherargs, runargs, comm):
             toast.ops.Reset(detdata=[defaults.det_data])
         # Append a weather model
         wrk.append_weather_model(job, otherargs, runargs, data)
-
+        # Before running simulations based on real data, we need
+        # a starting noise estimate
+        wrk.diff_noise_estimation(job, otherargs, runargs, data)
```

Comment:
This needs to happen before zeroing the loaded data.

Reply:
Done, and the load-or-simulate logic is now extracted as we discussed offline.